README.md

```diff
@@ -3,7 +3,14 @@ The beginnings of a Sculpin-based style guide creation tool.
 
 ## Dependencies
 1. NodeJS 
+   1. __Recommended method__: To intall NodeJS using Homebrew run 'brew install node' 
```

Comment:
Also use this ( ` ) character to get the code formatting. E.g.: `brew install node` (key under escape on a mac keyboard)